docs/tutorials/upgrade.md

```diff
@@ -96,10 +96,9 @@ application:
   - Configuration changes:
     - `streaming.forceTransmuxTS` has been renamed to `streaming.forceTransmux`
       (deprecated in v4.3.0)
-    - Callbacks that used to accept `Element` now accept `shaka.externs.xml.Node`:
-      (`manifest.dash.manifestPreprocessor` and `manifest.mss.manifestPreprocessor`).
-      `getAttribute()` and `textContent` results must now be decoded if they might contain
-      escape sequences. You can use `shaka.util.StringUtils.htmlUnescape` for this purpose.
+    - `manifest.dash.manifestPreprocessor` and `manifest.mss.manifestPreprocessor`
+      have been replaced with `manifest.dash.manifestPreprocessorTXml` and
+      `manifest.mss.manifestPreprocessorTXml` callbacks.
```

Comment:
I think this should still mention the difference in decoding requirements for the new versions.